lib/src/mistralai_client_dart_base.dart

```diff
@@ -66,3 +123,13 @@ class MistralAIClient {
     throw UnimplementedError('to be implemented');
   }
 }
+
+/// Simple exception class for Mistral AI Client exceptions.
+class MistralAIClientException implements Exception {
+  MistralAIClientException([this.message = '']);
+
+  final String message;
+
+  @override
+  String toString() => 'MistralAIClientException: $message';
+}
```

Comment:
You recreated `Exception` from JS `MistralAI Client`👏.

Reply:
Yup :)

Reply:
As it's a positive comment I will resolve it on behalf of @MarcinGruchala.